scirpy/tests/test_io.py

```diff
@@ -123,6 +124,60 @@ def test_airr_cell_empty():
     ac.to_scirpy_record()
 
 
+@pytest.mark.parametrize(
+    "chain_dict,expected",
+    [
+        (
+            {
+                "v_call": "nan",
+                "d_call": "nan",
+                "j_call": "nan",
+                "c_call": "nan",
+            },
+            None,
+        ),
+        (
+            {
+                "v_call": "TRAV36/DV7*01",
+                "d_call": "TRDJ2*01",
+                "j_call": "nan",
+                "c_call": "TRDC",
+            },
+            "TRD",
+        ),
+        (
+            {
+                "v_call": "TRAV36/DV7*01",
+                "d_call": "TRDJ2*01",
```

Comment:
this is a `j_call`?

Reply:
good catch! fixed...

Reply:
should probably fill them with a dummy `d_call` as well. like `TRDD1*01`